coordinator/points_writer.go

```diff
@@ -420,9 +513,9 @@ func (w *PointsWriter) WritePointsPrivileged(writeCtx tsdb.WriteContext, databas
 	w.Subscriber.Send(pts)
 	atomic.AddInt64(&w.stats.SubWriteOK, 1)
 
-	if err == nil && len(shardMappings.Dropped) > 0 {
-		err = tsdb.PartialWriteError{Reason: "points beyond retention policy or outside permissible write window",
```

Comment:
The new code would have an error message of "dropped %d points outside retention policy of duration". I assume this message is what the client will receive. Any risk of this breaking any client libraries or C1 alerting?

Reply:
In theory, it could break picky client code, but my hope is that people are switching on the 4XX HTTP error, not the message text.